packages/website/plugins/generated-rule-docs/spliceWhenNotToUseIt.ts

```diff
@@ -0,0 +1,64 @@
+import type * as mdast from 'mdast';
+import type * as unist from 'unist';
+
+import type { RequiredHeadingIndices } from './requiredHeadings';
+import { spliceChildrenAndAdjustHeadings } from './splicing';
+import type { RuleMetaDataWithDocs } from './types';
+
+export function spliceWhenNotToUseIt(
```

Comment:
Can we not call functions `spliceX`, `unshiftX`, or `pushX`? They both function as `insert` and which one is used is an implementation detail.

Reply:
😍 I love this comment so much. Yes!